k8s-daemonset/k8s/hello-world.yml

```diff
@@ -50,7 +50,7 @@ metadata:
 spec:
   selector:
     app: hello
-  type: LoadBalancer
+  type: ClusterIP
```

Comment:
Ah, ok, makes sense -- in that case can you make the same change in `hello-world-1_4.yml`?